guides/source/action_mailbox_basics.md

```diff
@@ -265,9 +267,15 @@ https://actionmailbox:PASSWORD@example.com/rails/action_mailbox/sendgrid/inbound
 
 NOTE: When configuring your SendGrid Inbound Parse webhook, be sure to check the box labeled **“Post the raw, full MIME message.”** Action Mailbox needs the raw MIME message to work.
 
-## Examples
+## Processing Incoming Email
+
+Processing incoming emails usually entails using the email content to create models, update views, queue background work, etc. in your Rails application.
+
+Before you can start processing incoming emails, you'll need to setup Action Mailbox routing and create mailboxes.
+
+### Configure routing
 
-Configure basic routing:
+Routes for Action Mailbox are added to the `application_mailbox.rb` file. The regular expression matches the incoming email's `to` field.
```

Comment:
Maybe we can expand a bit on what "routing" does / why it's needed? It might not be as clear to new people.

```suggestion
After an incoming email is received via the configured ingress, it needs to be forwarded to a mailbox for actual processing by your application. Routing in Action Mailbox is where we define which emails go to which mailboxes for processing. They are added to the `application_mailbox.rb` file. The regular expression matches the incoming email's `to` field:
```

Another idea, that makes the connection with the app routing system:

```suggestion
After an incoming email is received via the configured ingress, it needs to be forwarded to a mailbox for actual processing by your application. Much like the [Rails router that dispatches URLs to controllers](https://guides.rubyonrails.org/routing.html), routing in Action Mailbox defines which emails go to which mailboxes for processing. They are added to the `application_mailbox.rb` file. The regular expression matches the incoming email's `to` field:
```

These are just suggestions to give an idea, not necessarily final copy, so feel free to tweak as you see fit.